scripts/components/LviFavorites.ts

```diff
@@ -17,24 +18,31 @@ export default class LviFavorites extends LviFavoritesDesign {
         return this.lblFavoriteItemTitle.text;
     }
     set itemTitle(value: string) {
-        this.lblFavoriteItemTitle.text = value
+        this.lblFavoriteItemTitle.text = value;
     }
     get itemPrice(): any {
         return this.lblFavoriteItemPrice.text;
     }
     set itemPrice(value: any) {
-        this.lblFavoriteItemPrice.text = value
+        this.lblFavoriteItemPrice.text = value;
     }
     get itemImage(): any {
         return this.imgFavoriteItem.image;
     }
     set itemImage(value: any) {
-        this.imgFavoriteItem.image = Image.createFromFile(`images://${value}`)
+        this.imgFavoriteItem.image = Image.createFromFile(`images://${value}`);
     }
     get itemDesc(): string {
         return this.lblFavroiteItemDesc.text;
     }
     set itemDesc(value: string) {
-        this.lblFavroiteItemDesc.text = value
+        this.lblFavroiteItemDesc.text = value;
```

Comment:
Favroite
ItemDesc

we shouldn't be shortening names

Reply:
Done.